apps/snfoundry/contracts/src/test/test_cofi_collection.cairo

```diff
@@ -0,0 +1,418 @@
+use contracts::cofi_collection::ICofiCollectionDispatcher;
```

Comment:
I'd recommend to use the test directory structure as mentioned in [the documentation](https://foundry-rs.github.io/starknet-foundry/testing/test-collection.html#the-tests-directory)

Reply:
For some reason it stops running the tests if I move to this structure. We'll kee it for the moment but lets discuss about it.